jekyll/_docs/test-metadata.md

```diff
@@ -121,6 +122,16 @@ test:
 ### test2junit for Clojure tests
 You can use [test2junit](https://github.com/ruedigergad/test2junit) to convert Clojure test output to XML format. For more details, please checkout our [sample project](https://github.com/kimh/circleci-build-recipies).
 
+## Using the `files` modifier
+
+If you are collecting test metadata for a custom build step as documented above, you can
+still use runtime-based test splitting with the `files` [modifier](/docs/configuration/#modifiers).
```

Comment:
This link `[modifier](/docs/configuration/#modifiers)` needs to be changed to `[modifier]({{site.baseurl}}/configuration/#modifiers)`. This is due to relative links failing on RSS feeds as well as using the variable in case the docs location ever changes.